lib/process-services-cloud/src/lib/form/components/widgets/radio-buttons/radio-buttons-cloud.widget.spec.ts

```diff
@@ -74,6 +75,23 @@ describe('RadioButtonsCloudWidgetComponent', () => {
         expect(field.updateForm).toHaveBeenCalled();
     });
 
+    it('should call rest api only when url is present and field type is rest', () => {
+        const getFieldConfig = (optionType: string, restUrl: string) => new FormFieldModel(new FormModel(), { optionType, restUrl });
+        spyOn(formCloudService, 'getRestWidgetData').and.returnValue(of([]));
+
+        widget.field = getFieldConfig('manual', 'fake-rest-url');
```

Comment:
same as before

Reply:
answer same as before, I guess old project still can have this config manual + restUrl, so it's coverage for that